README.md

```diff
@@ -61,11 +61,11 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
+<!-- ## Acknowledgments
 
 - Hat tip to anyone whose code was used
 - Inspiration
-- etc
+- etc -->
```

Comment:
- [ ] Kindly remove commented code. If you do not have the necessary information now, you can add it at another time.